### PR TITLE
Update product designer exercise public description

### PR DESCRIPTION
**Pull request by rrhyne**

### CODE REVIEW

**Review by AlicjaSuska — APPROVED** [review]

File: handbook/product/roles/interviews/product_designer/pd_design_exercise.md [hunk 1]
```diff
@@ -40,16 +40,28 @@ Ask follow up questions:
...
+#### Please note: 
+
+The chosen problem has been implemented, so interviewers will need to role-play their involvement slightly, but our goal is to make the exercise as close to a collaborative working session as possible. This does not mean our solution is the right or best solution, and we are not evaluating the outcomes of this session against what we’ve done in the past.
+
+We are not providing the interview ahead of time to ensure all candidates have equal time to devote to the problem, regardless of external commitments. This means the session will be contrary to our typically async method of working, but we feel it is more important to respect the candidate’s time than to replicate our process exactly. 
+
+#### We will evaluate the following:
+
+* Can the candidate lead the room, keep the group focused, and facilitate the exercise.
+* What's the candidate process of exploring new problems and getting all the information necessary to complete the task
```
Comment (rrhyne) [inline handbook/product/roles/interviews/product_designer/pd_design_exercise.md:59]:
```suggestion
* How does the candidate explore new problems and get the information necessary to complete a task
```

**Comment by christinaforney** [comment]
@rrhyne @AlicjaSuska @quinnkeast - I made a few additional tweaks and changes to update a few things, correct timing, and to make sure to call out that an important part of this working session is to evaluate collaboration with PMs and engineering.

**Review by quinnkeast — APPROVED** [review]

**Comment by rrhyne** [comment]
Forgive me if this was discussed and I missed it, but I'm curious as to why the time for the exercise was reduced from 90 minutes to 60? I think this may be a tight squeeze now.

**Comment by christinaforney** [comment]
> Forgive me if this was discussed and I missed it, but I'm curious as to why the time for the exercise was reduced from 90 minutes to 60? I think this may be a tight squeeze now.

I discussed this with you, Juliana and Joel after the last exercise. We agreed 5 for intros, 15 min for part 1, 60 min for part 2, 10 for questions, total of 90 for the entire interview. This had been updated in the interviews docs but didn't get updated here. Should we revisit this decision?

**Comment by rrhyne** [comment]
Ah, my bad. It has been a few weeks now. Let's give it a go and see how it works out.

**Pull request merged**